src/js/navitron.js

```diff
@@ -135,7 +145,13 @@
                             $pane.find(selectors.CONTENT).attr('tabindex', 0).focus();
 
                             plugin._trigger('onShown', { pane: $pane });
-                        });
+                        };
+
+                        if (typeof window.requestAnimationFrame === 'function') {
```

Comment:
Not a bad thought @jeremywiebe . But it also occurred to me that while [requestAnimationFrame is pretty well supported](http://caniuse.com/#feat=requestanimationframe), Navitron's code shouldn't exclusively rely on it always being available hence why I conditionalized wherever it is used in the source code. (You can see that we're still targeting more legacy browser compatibility in the Readme)